### ISSUE TITLE
search result is empty if in the search bar type a dot

### ISSUE THREAD

**Issue by tjmaru** [padloc/padloc#753]
HI everyone,

I'm using self-hosted padloc version 4.2.0 in docker.
I found that if you are searching for something containing a dot, the result will be empty.
see the example:
1) search without dot
![Screenshot from 2024-06-16 17-24-30](https://github.com/padloc/padloc/assets/30507792/6a877d94-2019-4de0-83f2-e12d4fa13e97)
2) search with dot
![Screenshot from 2024-06-16 17-24-57](https://github.com/padloc/padloc/assets/30507792/6bd39ee9-d051-4461-8aba-0e996089e9e0)

is it intended or maybe I have to escape the dot somehow?

thank you for any info.

**Comment by Shikharishere** [comment]
Hey I tried the same thing and I found the same issue. I did this in the padloc's chrome extension. Maybe it is expected that we don't use dots in a username. 
Later I found out that same happens for the vault names. I think it expects us to use dots only in urls, emails etc.
Here are the screenshots.
![image](https://github.com/padloc/padloc/assets/71226935/6d62d331-3d81-45f4-9c1c-d66115de6651)
![image](https://github.com/padloc/padloc/assets/71226935/f6680a95-c8a9-4720-8cd3-62e56fb91ac5)

**Comment by tjmaru** [comment]
Hey @Shikharishere , thank you for testing and confirmation.
As I see the behaviour is the same for any type of fields. And the problem I see here is in case I search for the entity and write a subdomain URL I'm not able to find it as well.

**Comment by Shikharishere** [comment]
Yeah, @tjmaru that's correct. When you search for a vault the name is like a URL or subdomain URL. In that case, it doesn't search them at all. Although, when you're on a site it does find the vault related to it correctly.
